libraries/slave_windows.rb

```diff
@@ -81,9 +80,24 @@ def action_create
     end
 
     protected
-
+    
     # Embedded Resources
 
+    # Creates a `directory` resource that represents the directory
+    # specified the `remote_fs` attribute. The caller will need to call
+    # `run_action` on the resource.
+    #
+    # @return [Chef::Resource::Directory]
+    #
+    def remote_fs_dir_resource
+      return @remote_fs_dir_resource if @remote_fs_dir_resource
```

Comment:
This duplicates code from the parent class.  Could you put the offending "owner/group" statements in an "unless windows" clause - or even better, pull those out to a "remote_fs_dir_resource_perms" method.  This way the parent can do it's unixy permissions thing and you'd need to override just that method here to do the windows dance.